libretroshare/src/pqi/p3linkmgr.cc

```diff
@@ -1063,8 +1063,12 @@ void    p3LinkMgrIMPL::peerStatus(const RsPeerId& id, const pqiIpAddrSet &addrs,
 	uint32_t peer_vs_dht = 0;
 	uint32_t peerNetMode = 0;
```

Comment:
I thought your PR includes an enum for peerNetMode.

Reply:
Not yet, since the first batch got rather large already. 
I wanted to get this merged first before converting more (and making this pr even larger).